s3transfer/download.py

```diff
@@ -131,6 +131,27 @@ def _get_temp_fileobj(self):
         return f
 
 
+class DownloadSeekableOutputManager(DownloadOutputManager):
+    @classmethod
+    def is_compatible(self, download_target):
+        return (
+            hasattr(download_target, 'seek') and
+            hasattr(download_target, 'tell')
```

Comment:
Yeah that fine. I would just need to figure out how ``seekable()`` is implemented in python. I think it is written in C so it may be interesting.